Makefile

```diff
@@ -154,7 +154,7 @@ docker-start:
 docker-stop:
 	@docker stop `cat .docker-id`
 
-eunit: export BUILDDIR = $(shell pwd)
+eunit: export BUILDDIR = $(shell pwd) ERL_FLAGS = "-config `pwd`/tmp/etc/sys.config"
```

Comment:
I did the following [trick](https://github.com/apache/couchdb/pull/350/files#diff-b67911656ef5d18c4ae36cb6741b7965R158) for multi exports. So far it works, but cannot say for sure if that's how they should be defined in Makefiles.